tests/View/ViewBladeCompilerTest.php

```diff
@@ -814,4 +814,28 @@ public function testGetTagsProvider()
             ['(((', ')))'],
         ];
     }
+
+    /**
+     * @expectedException \RuntimeException
```

Comment:
Being able to support things like `@foreach $messages => $message` would be highly backward incompatible, hence I cannot do better... The above call would simple raise an exception saying that the `foreach` expression is _empty_.